templates/teable/meta.yaml

```diff
@@ -0,0 +1,83 @@
+name: Teable
+description:
+  Teable is a powerful, self-hosted service designed for managing team
+  collaboration and projects efficiently. It offers a clean and user-friendly
+  interface, supports plugins for extended functionality, and provides detailed
+  analytics on team activities. With Teable, you can maintain full control over
+  your data and workflows.
+
+instructions:
+  Wait for the migration to complete, then start using Teable for managing your
+  projects.
+
+changeLog:
+  - date: 2025-02-12
+    description: Initial Template Release
+
+links:
+  - label: Documentation
+    url: https://teable.io/docs
+  - label: Github
+    url: https://github.com/teableio/teable
+
+contributors:
+  - name: Ahson Shaikh
+    url: https://github.com/Ahson-Shaikh
+
+schema:
+  type: object
+  required:
+    - appServiceName
+    - appServiceImage
+    - migrationServiceImage
+  properties:
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: teable
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: ghcr.io/teableio/teable:latest
```

Comment:
Let's not use latest

Also, you should add a checkbox for arm64. If checked, add a `-arm64` at the end of the image.